packages/dnb-eufemia/src/components/autocomplete/Autocomplete.js

```diff
@@ -1353,7 +1353,7 @@ class AutocompleteInstance extends React.PureComponent {
         .filter(({ word, wordIndex }) => {
           if (searchNumbers) {
             // Remove all other chars, except numbers, so we can compare
-            word = word.replace(/[^\d\w]/g, '')
+            word = word.replace(/[^\d\wÆØÅæøå]/g, '')
```

Comment:
```suggestion
            word = word.replace(/[^\p{L}\p{N}]+/gu, '')
```

- [^\p{L}\p{N}] means "any character not a letter or digit."
- u flag makes it a Unicode-aware search, and allows you to use \p{...} property escapes.

Reply:
TIL! Cool 😊 
- 📚 [Unicode mode (`u`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/unicode)
- 📚 [Unicode character class escape](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape)
- "L" and "N" are "General_Category" property values - they are [listed on the Unicode website](https://unicode.org/reports/tr18/#General_Category_Property)
  - From what I read here, this regex could also be expressed using the long form names "Letter" and "Number" (which might be easier to read, but could also just be unfamiliarity?)

Reply:
The comment above this line seems a little outdated now, by the way.